serverless/pages/search-your-data-semantic-search.asciidoc

```diff
@@ -6,140 +6,48 @@
-the original search terms in a training dataset, and weights to control how
-important each is.
+[[elasticsearch-reference-semantic-search-semantic-text]]
+== Semantic search with `semantic text`
 
-[discrete]
-[[semantic-search-deploy-nlp-model]]
-== Deploy the model
-
-After you decide which model you want to use for implementing semantic search,
-you need to deploy the model in {es}.
+The `semantic_text` field simplifies semantic search by providing inference at ingestion time with sensible default values, eliminating the need for complex configurations. 
 
-include::../partials/deploy-nlp-model-widget.asciidoc[]
+Learn how to implement semantic search with `semantic text` in the https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search-semantic-text.html[Elasticsearch docs →].
 
 [discrete]
-[[semantic-search-field-mappings]]
-== Map a field for the text embeddings
+[[elasticsearch-reference-semantic-search-inference-api]]
+== Semantic search with the inference API
 
-Before you start using the deployed model to generate embeddings based on your
-input text, you need to prepare your index mapping first. The mapping of the
-index depends on the type of model.
+The inference API workflow enables you to perform semantic search using models from a variety of services, such as Cohere, OpenAI, HuggingFace, Azure AI Studio, and more. 
 
-include::../partials/field-mappings-widget.asciidoc[]
-
-[discrete]
-[[semantic-search-generate-embeddings]]
-== Generate text embeddings
+Learn how to implement semantic search with the inference API in the {ref}/semantic-search-inference.html[Elasticsearch docs →].
 
-Once you have created the mappings for the index, you can generate text
-embeddings from your input text. This can be done by using an
-{ref}/ingest.html[ingest pipeline] with an {ref}/inference-processor.html[inference processor].
-The ingest pipeline processes the input data and indexes it into the destination
-index. At index time, the inference ingest processor uses the trained model to
-infer against the data ingested through the pipeline. After you created the
-ingest pipeline with the inference processor, you can ingest your data through
-it to generate the model output.
-
-include::../partials/generate-embeddings-widget.asciidoc[]
-
-Now it is time to perform semantic search!
 
 [discrete]
-[[semantic-search-search]]
-== Search the data
-
-Depending on the type of model you have deployed, you can query sparse vectors
-with a sparse vector query, or dense vectors with a kNN search.
+[[elasticsearch-reference-semantic-search-elser]]
+== Semantic search with ELSER
 
-include::../partials/search-widget.asciidoc[]
+Elastic Learned Sparse EncodeR - or ELSER - is an NLP model trained by Elastic
+that enables you to perform semantic search by using sparse vector
+representation. Instead of literal matching on search terms, semantic search
+retrieves results based on the intent and the contextual meaning of a search
+query.
```

Comment:
hould be "Model deployment workflow" and should point [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search-deployed-nlp-model.html)